src/components/collection_voting_slab/DatasetsRequestedPanel.js

```diff
@@ -0,0 +1,141 @@
+import {a, div, h, span} from "react-hyperscript-helpers";
+import {useCallback, useEffect, useState} from "react";
+import {isNil, filter, includes, map} from "lodash/fp";
+
+const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: '500',
+    color: '#333F52',
+    backgroundColor: '#F1EDE8',
+    borderRadius: '0 0 4px 4px',
+    borderBottom: '4px #646464 solid',
+    padding: '15px 25px',
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '1.5rem'
+  },
+  heading: {
+    fontWeight: 'bold',
+    display: 'flex',
+    columnGap: '0.5rem',
+    alignItems: 'center'
+  },
+  datasetCount: {
+    color: '#747474',
+    fontSize: '1.2rem'
+  },
+  datasetList: {
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '0.75rem'
+  },
+  link: {
+    color: '#0948B7',
+    fontWeight: '500',
+  },
+  skeletonLoader: {
+    height: '30px',
+    width: '60%'
+  }
+};
+
+export default function DatasetsRequestedPanel(props) {
+  const [filteredDatasets, setFilteredDatasets] = useState([]);
+  const [visibleDatasets, setVisibleDatasets] = useState([]);
+  const [datasetCount, setDatasetCount] = useState(0);
+  const [expanded, setExpanded] = useState(false);
+  const collapsedDatasetCapacity = 5;
+  const {bucketDatasetIds, dacDatasetIds, collectionDatasets, isLoading} = props;
+
+
+  const requestedDatasetIds = useCallback(() => {
+    const datasetsForDacInBucket =  filter(bucketDatasetId => {
+      return includes(bucketDatasetId)(dacDatasetIds);
+    })(bucketDatasetIds);
+
+    return filter(dataset => {
+      return includes(dataset.datasetId)(datasetsForDacInBucket);
```

Comment:
The dataset id property on the dataset is weirdly capitalized due to history ... this line should be:
```suggestion
      return includes(dataset.dataSetId)(datasetsForDacInBucket);
```

Reply:
Just an FYI for testing purposes, I added this component to the DarCollectionReview page and viewed collection id 190 on dev:
```
      h(DatasetsRequestedPanel, {
        bucketDatasetIds: [1250, 449],
        dacDatasetIds: [1250],
        collectionDatasets: collection.datasets,
        isLoading
      }),
```
Fiddling with the bucketDatasetIds and dacDatasetIds shows me the results I was expecting to see.